package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "automail",
+  "version": "10.0.0",
```

Comment:
Is having only the major/minor version number here an option?

Reply:
I think it follows semantic versioning just like userscripts
https://docs.npmjs.com/cli/v8/configuring-npm/package-json#version

so major.minor.revision is required